src/layers/core/point-density-grid-layer/grid-aggregator.js

```diff
@@ -0,0 +1,112 @@
+const R_EARTH = 6378;
```

Comment:
My vision here is that we would use the `Viewport` classes to help us define a grid in any coordinate mode (lng/lat, meter offsets, identity). I realize that would take a bit of work. In the mean time, trying to avoid tying variable names to lng/lat/kilometers etc would be a good first step.